docs/reference/builder.md

```diff
@@ -1484,7 +1483,7 @@ will require application source code to be added in a particular
 directory, and it might require a build script to be called *after*
 that. You can't just call `ADD` and `RUN` now, because you don't yet
 have access to the application source code, and it will be different for
-each application build. You could simply provide application developers
+each application builds. You could simply provide application developers
```

Comment:
This is incorrect, this refers to "a build" of the application (i.e. a "release" or an "artifact" that is produced)

please change back to what it was